pom.xml

```diff
@@ -188,6 +188,13 @@
             <groupId>us.springett</groupId>
             <artifactId>alpine-server</artifactId>
             <version>${lib.alpine.version}</version>
+            <exclusions>
+                <!-- this exclusion done to get rid of vulnerability in the project as this transitive dependency is not being used in the project -->
```

Comment:
Could you include *which* vulnerabilities this is supposed to fix?

Reply:
done